test/edu-benefits/5490/config/benefitSelection.unit.spec.jsx

```diff
@@ -28,15 +38,31 @@ describe('Edu 5490 benefitSelection', () => {
     const form = ReactTestUtils.renderIntoDocument(
       <DefinitionTester
           schema={schema}
-          data={{ benefit: 'chapter35' }}
+          data={{}}
+          state={{
+            benefitSelection: {
+              uiSchema: {
+                benefit: {
+                  'ui:options': {
+                  }
+                }
+              }
+            }
+          }}
           uiSchema={uiSchema}/>
     );
     const formDOM = findDOMNode(form);
 
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_benefit_0'), {
+      target: {
+        checked: true
+      }
+    });
     // check that an alert box is shown below the chapter35 option
-    expect(formDOM
-      .querySelector('.form-radio-buttons + span .usa-alert')
-    ).not.to.be.null;
+    setTimeout(() => {
```

Comment:
We're starting to disable too many tests due to timing out, so I'll spend a bit more time seeing if I can trick this into showing up without a timeout. Otherwise, will disable again, but this isn't sustainable.